docs/app/content-collections.md

```diff
@@ -1,168 +1,123 @@
 
-## Filtering Items
+- [Table Layout](/app/layouts/#table-layout)
+- [Card Layout](/app/layouts/#card-layout)
 
-**[Learn more about Filtering](/configuration/filter-rules/).**
+::: tip
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to filter
-3. Click on Filter (filter icon) located in the header
-4. Click the "Add Filter" dropdown and select the field to filter on
-5. Adjust the operator (eg: "Contains") as needed by clicking on it
-6. Enter a value in the field filter's input(s)
-7. Remove unwanted filters by hovering over the field and clicking "X"
+Automatic sorting is a non-destructive action which does not change your data.
 
-## Viewing Archived Items
+:::
 
-**[Learn more about Archive](/configuration/data-model/#archive).**
+## View Archived Items
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to view
-3. Click "Archive" in the page sidebar
-4. Choose the desired view: `Show Items` (default), `Show Archived Items`, or `Show Items + Archived Items`
+<video autoplay muted loop controls title="View Archived Items">
+	<source src="https://cdn.directus.io/docs/v9/app-guide/content/content-collections/content-collections-20220415A/view-archived-items.mp4" type="video/mp4" />
+</video>
 
-## Bookmarking Item Presets
+To view an archived Item, follow the steps below. [Learn More](/app/content-items/#archive-an-item)
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to bookmark
-3. Configure the page as desired (eg: layout, options, search, filters, order, etc)
-4. Click the "Create Bookmark" (bookmark icon) button in the header beside the page title
-5. Rename bookmarks by right-clicking them in the navigation sidebar and selecting "Rename Bookmark"
-6. Delete bookmarks by right-clicking them in the navigation sidebar and selecting "Delete Bookmark"
+1. Navigate to the Content Module and select the desired Collection.
+2. Click **"Archive"** in the Sidebar.
+3. Choose the desired view:
+   - **"Show Items"**
+   - **"Show Archived Items"**
+   - **"Show Items + Archived Items"**
 
-## Importing / Exporting Items
+## Create Presets and Bookmarks
 
-### Import
+<video autoplay muted loop controls title="Create Presets and Bookmarks">
+	<source src="https://cdn.directus.io/docs/v9/app-guide/content/content-collections/content-collections-20220415A/presets-and-bookmarks-20220415A.mp4" type="video/mp4" />
+</video>
 
-**[Learn more about Import Data from a File](/reference/system/utilities/#import-data-from-file)**
+Save how a Collection Page looks so that you can return to the exact same view later, including details such as the
+Layout, searches, filters, sorting, ordering, etc. [Learn More](/configuration/presets-bookmarks/)
```

Comment:
@benhaynes Specifically asked me to start using [Learn more]() on all end links.
Let me know what you guys decide you want.

Reply:
True! But Rijk is also right here. The most important thing is that we're consistent. What I want to avoid is long links within the docs. As long as we don't make the whole sentence clickable (just the relevant words) then I'm OK if Rijk wants us to make this switch.

Reply:
I do! Much better for a11y and SEO. Re-using the page title is often an easy one `To learn more about X, see our guide: ["How to Y"](#)`. Alternatively, we can inline it (as long as the relevant descriptive part is highlighted) `To learn more, see [our guide on Y](#) and [our other doc on the topic](#)`.